holidays/countries/greenland.py

```diff
@@ -75,7 +75,7 @@ def _populate_optional_holidays(self):
         # Epiphany.
         self._add_epiphany_day(tr("Mitaarneq"))
 
-        # International Workers' Day.
+        # Workers' Day.
```

Comment:
I don't mind simplifying, just checked Wikipedia -- it has `Intertnational` in the name https://en.wikipedia.org/wiki/International_Workers%27_Day

Reply:
Hmm, looks like first I need to review Scandinavian holidays entirely.

Reply:
Although Greenlandic name for this holiday is "Sulisartut ulluat" (and there is no word “international”), the translation into all Scandinavian languages includes this word. So let's leave it in the English version as well. 🤷‍♂️